onnxruntime/test/providers/cpu/tensor/gather_op_test.cc

```diff
@@ -32,8 +32,11 @@ TEST(GatherOpTest, Gather_axis0) {
     test.Run();
   };
 
-  run_test(false);
+#if defined(USE_NNAPI)
```

Comment:
do we need these ifdefs now that the op support checker will skip Gathers with non-int32 indices?

Reply:
no we don't. removed them. thx for the catch up.